common.go

```diff
@@ -600,6 +601,9 @@ type Config struct {
 	// Allow short headers (experimental, and only applies to server).
 	AllowShortHeaders bool
 
+	//If true disables use of the Extended Master Secret extension
```

Comment:
`gofmt`, or maybe space after `//` is not enforced but definitely required by upstream (and me 😇). Here and other places.